src/Datadog.Trace/Util/ProcessHelpers.cs

```diff
@@ -1,4 +1,6 @@
+using System;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 
 namespace Datadog.Trace.Util
 {
```

Comment:
Thanks for the additional Process helper! Can we also cache the first `Process.GetCurrentProcess()` result in a static field so we don't have to repeatedly call it? It means we would also need to dispose it when the static `_runtimeMetricsWriter` instance is disposed

Reply:
I don't know. If I cache the instance in RuntimeMetricsWriter, then won't we run into the same issues that made you add ProcessHelpers in the first place?

And if I cache the instance in ProcessHelpers then we need to add some synchronization (might be OK though, since it's very unlikely to be contended)

Reply:
I cached the process in ProcessHelpers, is that OK?

Reply:
Looks good 👍